--- icoutils-0.18.0/extresso/Makefile.am~	Wed Feb 19 00:49:05 2003
+++ icoutils-0.18.0/extresso/Makefile.am	Wed Feb 19 00:49:05 2003
@@ -1,11 +1,12 @@
-noinst_MANS = \
+man_MANS = \
   extresso.1 \
   genresscript.1
 
 EXTRA_DIST = \
-  $(noinst_MANS) \
+  $(man_MANS) \
   extresso.in \
   genresscript.in
 
-all-local:
-	chmod a+x extresso genresscript
+bin_SCRIPTS = \
+  extresso \
+  genresscript
