--- icoutils-0.16.0/compat/Makefile.am.orig	Wed Oct 31 13:41:45 2001
+++ icoutils-0.16.0/compat/Makefile.am	Tue Oct 22 16:29:54 2002
@@ -4,18 +4,8 @@
 	compat.h \
 	dirname.c \
 	dirname.h \
-	getline.c \
-	getline.h \
 	getstr.c \
 	getstr.h \
 	memrchr.c \
 	programname.c \
-	programname.h \
-	strndup.c \
-	strnlen.c \
-	strtol.c \
-	strtoll.c \
-	strtoul.c \
-	strtoull.c \
-	vasprintf.c \
-	vasprintf.h
+	programname.h
